diff --git a/src/lib.rs b/src/lib.rs
index aa06883..5eaa524 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -58,13 +58,14 @@ pub mod memory;
 pub(crate) mod migration;
 pub(crate) mod multimodal;
 pub mod observability;
-pub(crate) mod onboard;
+pub mod onboard;
 pub mod peripherals;
 pub mod providers;
 pub mod rag;
 pub mod runtime;
 pub(crate) mod security;
 pub(crate) mod service;
+pub mod setup;
 pub(crate) mod skills;
 pub mod tools;
 pub(crate) mod tunnel;
